presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestJdbcPreparedStatement.java

```diff
@@ -0,0 +1,190 @@
+                assertTrue(rs.next());
+                assertTestValues(rs);
+                assertFalse(rs.next());
+            }
+        }
+    }
+
+    @Test
+    public void testExecuteUpdate()
+            throws Exception
+    {
+        try (Connection connection = createConnection("blackhole", "blackhole")) {
+            try (Statement statement = connection.createStatement()) {
+                statement.execute("CREATE TABLE test_execute_update (" +
+                        "c_boolean boolean, " +
+                        "c_tinyint tinyint, " +
+                        "c_smallint smallint, " +
+                        "c_integer integer, " +
+                        "c_bigint bigint, " +
+                        "c_real real, " +
+                        "c_double double, " +
+                        "c_decimal decimal, " +
+                        "c_varchar varchar, " +
+                        "c_varbinary varbinary, " +
+                        "c_date date, " +
+                        "c_time time, " +
+                        "c_timestamp timestamp," +
+                        "c_null bigint)");
+            }
+
+            try (PreparedStatement statement = connection.prepareStatement(
+                    "INSERT INTO test_execute_update VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)")) {
+                setTestValues(statement);
+                assertEquals(statement.executeUpdate(), 1);
+            }
+        }
+    }
+
+    @Test
+    public void testPrepareMultiple()
+            throws Exception
+    {
+        try (Connection connection = createConnection();
+                PreparedStatement statement1 = connection.prepareStatement("SELECT 123");
+                PreparedStatement statement2 = connection.prepareStatement("SELECT 456")) {
+            try (ResultSet rs = statement1.executeQuery()) {
+                assertTrue(rs.next());
+                assertEquals(rs.getLong(1), 123);
+                assertFalse(rs.next());
+            }
+
+            try (ResultSet rs = statement2.executeQuery()) {
+                assertTrue(rs.next());
+                assertEquals(rs.getLong(1), 456);
+                assertFalse(rs.next());
+            }
+        }
+    }
+
+    private static void setTestValues(PreparedStatement statement)
+            throws SQLException
+    {
+        statement.setBoolean(1, true);
+        statement.setByte(2, (byte) 2);
+        statement.setShort(3, (short) 3);
+        statement.setInt(4, 4);
+        statement.setLong(5, 5L);
+        statement.setFloat(6, 6.0f);
+        statement.setDouble(7, 7.0d);
+        statement.setBigDecimal(8, BigDecimal.valueOf(8L));
+        statement.setString(9, "nine'nine");
+        statement.setBytes(10, "ten\0ten".getBytes(UTF_8));
+        statement.setDate(11, new Date(11));
+        statement.setTime(12, new Time(12));
+        statement.setTimestamp(13, new Timestamp(13));
+        statement.setNull(14, Types.BIGINT);
```

Comment:
Why not `new BigInteger("14")`?

as far as nulls are concerned, they should be covered for other types too. And it would be good to check that
```
prepare ("select ?")
setNull(1, Types.BIGINT)
rs = execute
assert rs.getMetaData().getColumnType(1) == Types.BIGINT 
```

Reply:
Done